pubsub/google/cloud/pubsub/_http.py

```diff
@@ -203,9 +204,10 @@ def topic_publish(self, topic_path, messages):
         :rtype: list of string
         :returns: list of opaque IDs for published messages.
         """
-        _transform_messages_base64(messages, _base64_unicode)
+        messages_to_send = copy.deepcopy(messages)
```

Comment:
So I was just changing `_transform_messages_base64` and I realized it's used in `subscription_pull()` as well. 
We might want to edit the messages in place instead of copying for *pull* because it should increase the performance?

So I propose the current changes to be used.